package.yaml

```diff
@@ -1,5 +1,5 @@
 name:                fakedata-quickcheck
-version:             0.2.0
+version:             0.1.0
```

Comment:
We have a 0.2 release already made to hackage with minor documentation fixes.

Reply:
Got it, sorry about that.